pyproject.toml

```diff
@@ -76,6 +76,7 @@ all = [
     "fastapi",
     "h5netcdf",
     "h5py",
+    "hdf5plugin",
```

Comment:
Would you add this to the `server` block also? In server environments we sometimes just install `pip install "tiled[server]"` to avoid installing the client-only dependencies.

Reply:
sure, one second..

Reply:
added it to both other places where h5py was mentioned, in server and formats.

Reply:
Noticed I'd accidentally included a fix that was supposed to be in the other branch.